netlify.toml

```
@@ -8,7 +8,7 @@ command = "git submodule update --init --recursive --depth 1 && make non-product
 
 [build.environment]
 NODE_VERSION = "20.11.0"
```

We need to update this to resolve a conflict that arose after #47618 was merged.
```suggestion
NODE_VERSION = "20.17.0"
```